dali/python/nvidia/dali/pipeline.py

```diff
@@ -51,10 +51,13 @@ class Pipeline(object):
 Parameters
 ----------
 `batch_size` : int, optional, default = -1
-    Batch size of the pipeline. Negative values for this parameter
+    Maximum batch size of the pipeline. Negative values for this parameter
     are invalid - the default value may only be used with
     serialized pipeline (the value stored in serialized pipeline
-    is used instead).
+    is used instead). In most cases, the actual batch size of the pipeline
+    will be equal to the maximum one. Running DALI pipeline with batch size
```

Comment:
I got confused reading this. How about (just a suggestion):
```
Running the DALI Pipeline with a smaller batch size is also supported. The batch size might change from iteration to iteration.
```

Reply:
Done